### PR TITLE
Add default value to documented constructor parameters

### PR DESCRIPTION
**Pull request by RaananW**
When compiling babylon using closure compiler it complains that default parameters don't have the default value, in accordance to JSDoc specifications. as we are using tsdoc and not jsdoc, this introduces a bit of an issue if we document the constructor with default values (tsdoc doesn't have a definition for default value of a paremeter).

The solution is to add the default value to the actual documentation of the paremeter, if provided. This is needed if a constructor has a public member that is documented and has a default value. These are the ones I found, I guess there might be more and I will try adding a step to check for missing values in the future.

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15218/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/15218/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/15218/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15218/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15218/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15218/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15218/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15218/merge#BCU1XR#0

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/15218/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/15218/merge/testResults/webgpuplaywright/index.html

File: packages/dev/core/src/XR/features/WebXRBackgroundRemover.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ export class WebXRBackgroundRemover extends WebXRAbstractFeature {
     constructor(
         _xrSessionManager: WebXRSessionManager,
         /**
-         * read-only options to be used in this module
+         * [Object] read-only options to be used in this module
```
Comment (Popov72) [inline packages/dev/core/src/XR/features/WebXRBackgroundRemover.ts:63]: Is it not "Empty object"? Same comment x2 below.

**Review by sebavan — APPROVED** [review]

File: packages/dev/core/src/Physics/v1/physicsImpostor.ts [hunk 2]
```diff
@@ -443,7 +443,7 @@ export class PhysicsImpostor {
          */
         public object: IPhysicsEnabledObject,
         /**
-         * The type of the physics imposter
+         * [Object] The type of the physics imposter
```
[the remark this answers is not in the document]
Comment (RaananW) [reply packages/dev/core/src/Physics/v1/physicsImpostor.ts:446]: this is actually not needed. There is no default value. I guess I saw the line after when editing this. I'll remove this one.

Comment (RaananW) [inline packages/dev/core/src/Physics/v1/physicsImpostor.ts:446]:
```suggestion
         * The type of the physics imposter
```

File: packages/dev/core/src/XR/features/WebXRBackgroundRemover.ts (hunk 1 above)
Comment (Popov72) [reply packages/dev/core/src/XR/features/WebXRBackgroundRemover.ts:63]: In fact, I meant to use "[Empty object]" instead of "[Object]": is it not allowed?

Comment (RaananW) [reply packages/dev/core/src/XR/features/WebXRBackgroundRemover.ts:63]: Everything is allowed, it's just a string (in our case, not in JSDoc). I don't mind changing it

**Pull request merged**